graph/hasa.go

```diff
@@ -244,10 +253,18 @@ func (it *hasAContains) nextContains(ctx context.Context) bool {
 	for it.results.Next(ctx) {
 		link := it.results.Result()
 		if clog.V(4) {
-			clog.Infof("Quad is %v", it.qs.Quad(link))
+			qlv, err := it.qs.Quad(link)
```

Comment:
It ignores error here

Reply:
Previously if there was an error it would print "Quad is nil"

Reply:
added logging